.eslintrc

```diff
@@ -30,6 +29,13 @@
   "rules": {
     "@typescript-eslint/no-unused-vars": ["error"],
     "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-misused-promises": [
```

Comment:
got stuck figuring this out for a long time. can't figure it out

Reply:
I think I put that there out of rage. I'll remove it and see what gets called out.